ambry-store/src/main/java/com.github.ambry.store/StorageManager.java

```diff
@@ -202,11 +202,14 @@ DiskManager getDiskManager(PartitionId id) {
   }
 
   /**
-   * Check if a certain disk is available.
+   * Check if a certain disk is available at specific mount path
```

Comment:
Not sure what we mean by this comment. We are only passing the Disk object.

Reply:
Sorry, I forgot to update this. Will fix